server/etcdserver/server.go

```diff
@@ -1104,6 +1104,14 @@ func verifySnapshotIndex(snapshot raftpb.Snapshot, cindex uint64) {
 	})
 }
 
+func verifyConsistentIndexIsLatest(lg *zap.Logger, snapshot raftpb.Snapshot, cindex uint64) {
```

Comment:
Minor comment: we should consider to make this function has the same signature as `verifySnapshotIndex`.